pom.xml

```diff
@@ -31,651 +32,651 @@
       </dependency>
       <dependency>
         <groupId>org.nuxeo.template.rendering</groupId>
         <artifactId>nuxeo-template-rendering-samples</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.template.rendering</groupId>
         <artifactId>nuxeo-template-rendering-samples-jsf</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.theme</groupId>
         <artifactId>nuxeo-theme-migration</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.ecm.platform</groupId>
         <artifactId>nuxeo-tree-snapshot-jsf</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.ecm.platform</groupId>
         <artifactId>nuxeo-user-center</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.ecm.platform</groupId>
         <artifactId>nuxeo-user-dashboard</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.ecm.platform</groupId>
         <artifactId>nuxeo-user-dashboard-jsf</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.ecm.platform</groupId>
         <artifactId>nuxeo-user-notification</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.ecm.platform</groupId>
         <artifactId>nuxeo-user-oauth</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.ecm.platform</groupId>
         <artifactId>nuxeo-user-profile-jsf</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
       <dependency>
         <groupId>org.nuxeo.ecm.platform</groupId>
         <artifactId>nuxeo-web-resources-jsf</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
       </dependency>
 
       <!-- distribution -->
       <dependency>
         <groupId>org.nuxeo.ecm.distribution</groupId>
         <artifactId>nuxeo-distribution</artifactId>
-        <version>${project.version}</version>
+        <version>${nuxeo.jsf.version}</version>
```

Comment:
I wonder if it's relevant to have `nuxeo.jsf.version` as version for the `org.nuxeo.ecm.distribution` artifacts?
In fact, this particular `<artifactId>nuxeo-distribution</artifactId>` pom import might  not be needed anymore.

Reply:
You're right, the whole dependency can be removed.